web/templates/pages/members.html

```diff
@@ -20,13 +20,13 @@
 			<ul class="about_list">
 			{% for member in members_list %}
 				{% if member.activity_status == 'MC' %}
-					<li class="mentor-cat"><a href="{{member.githubuser}}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
+					<li class="mentor-cat"><a href="{% url 'web.views.member_page' user_id=member.id %}"><img src="../static/img/{{member.profile_picture}}">{{member.fullname}}</a> (Mentor cat)<li>
```

Comment:
What happens if I change the name of the `static` folder? You need to link static file trough `{% static %}` tag, that is why you loaded staticfiles at the beginning of file.